alert_group_nl_log2slack.py

```diff
@@ -33,6 +33,8 @@
 
 SLACK_USERMAP = {'alice': 'U0H87MYTC', 'frank': 'U025CBXTP'}
 
+HEALTH_FILE = os.environ.get('HEALTH_FILE', '')
```

Comment:
What happens when this is empty?

Opening (or touching) the empty file will likely fail and we'll crash. This should be handled more gracefully.

Reply:
In my latest commit if this isn't defined it will just ignore it and run without a `HEALTH_FILE`